doc/doxygen/src/css/riot.css

```diff
@@ -488,6 +488,10 @@ dl.todo > dt {
 }
 .params .paramdir {
   font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
+  padding-right: 0.5em;
+}
+.params .paramname {
+  padding-right: 0.5em;
```

Comment:
IMHO making it `1.5em` or `2em` looks even better ... definitely a bit larger than for `paramdir` above